[2025-02-18T04:32:25.748+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:32:25.770+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-11-23T00:00:00+00:00 [queued]>
[2025-02-18T04:32:25.779+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-11-23T00:00:00+00:00 [queued]>
[2025-02-18T04:32:25.780+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:32:25.794+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-11-23 00:00:00+00:00
[2025-02-18T04:32:25.804+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-11-23T00:00:00+00:00', '--job-id', '282', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpnjljaite']
[2025-02-18T04:32:25.806+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1092) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:32:25.807+0000] {standard_task_runner.py:105} INFO - Job 282: Subtask hello_task
[2025-02-18T04:32:25.807+0000] {standard_task_runner.py:72} INFO - Started process 1101 to run task
[2025-02-18T04:32:25.828+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:32:25.864+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-11-23T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:32:25.865+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:32:25.926+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-23T00:00:00+00:00'
[2025-02-18T04:32:25.927+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:32:25.936+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:32:25.936+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:32:25.943+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:32:25.945+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-11-23T00:00:00+00:00, execution_date=20241123T000000, start_date=20250218T043225, end_date=20250218T043225
[2025-02-18T04:32:25.996+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:32:26.004+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
